dev-guide/ci/events-files.md

```diff
@@ -0,0 +1,64 @@
+  * Tip: wait several seconds for the html page to finish rendering so you can use the text entry boxes to search
+
+* events.json (same as `oc_cmds/events` mentioned below except as json)
+  * generated via `oc get events --all-namespaces -o json” (see [source code](https://github.com/openshift/release/blob/f5017d5136a740a4186477b02bed70047ade200b/ci-operator/step-registry/gather/extra/gather-extra-commands.sh#L61))
+  * located in the prow job stored artifacts at `.../gather-extra/aftifacts/`
+  * [Example](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/gather-extra/artifacts/events.json)
+
+ * events (same as `gather-extra/artifacts/events.json` except not json)
+   * generated via `oc get events --all-namespaces` (see [source code](https://github.com/openshift/release/blob/f5017d5136a740a4186477b02bed70047ade200b/ci-operator/step-registry/gather/extra/gather-extra-commands.sh#L62))
+   * located in the prow job stored artifacts at `.../gahter-extra/artifacts/oc_cmds/events`
+   * [Example](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/gather-extra/artifacts/oc_cmds/events)
+
+* resource-events_(date)-(number).zip
+  * located in the prow job stored artifacts at `.../<test_name>/openshift-e2e-test/artifacts/junit`; there are two of them (not sure why?)
+    * One of the files contains events related to openshift-operator namespaces
+    * The other file contains events related to e2e
+  * Contains a watch of all events including "delete" related events
+  * [Example1](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/openshift-e2e-test/artifacts/junit/resource-events_20220321-040307.zip), [Example2](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-gcp-ovn-upgrade/1505747388038385664/artifacts/e2e-gcp-ovn-upgrade/openshift-e2e-test/artifacts/junit/resource-events_20220321-052002.zip)
+  * Here's a sample of the contents:
+
+```
+    ./default:
+    total 332
+    -rw-rw-r-- 1 dperique dperique 339013 Dec 31  1979 events.json  <-- why the ancient date?
+
+    ./e2e-check-for-dns-availability-1546:
+    total 68
+    -rw-rw-r-- 1 dperique dperique 67899 Dec 31  1979 events.json
+
+    ./e2e-k8s-service-load-balancer-with-pdb-new-485:
+    total 88
+    -rw-rw-r-- 1 dperique dperique 86468 Dec 31  1979 events.json
+```
+
+* e2e-events_(date)-(number).json
```

Comment:
there are monitor events, not API events.   the monitor is part of the openshift-tests invocation.  They do not have the same info as the API events.

Reply:
Would it be worth the bother of renaming these? I believe this is fairly confusing for anyone who encounters them.

Reply:
>Would it be worth the bother of renaming these? I believe this is fairly confusing for anyone who encounters them.

I'd merge it.  I'd also merge a PR to write the raw events separate from the computed ones.